maintenance_page/manifests/maintenance/deployment_maintenance.yml.tmpl

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: teacher-success-maintenance
```

Comment:
the service name is teach so this should be teach-maintenance (and the same elsewhere)